tools/amp_stt/tag_words.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+from pathlib import Path
+import json
+import logging
+import sys
+import os
+sys.path.insert(0, os.path.abspath('../../../../../tools/amp_schema'))
+from speech_to_text import SpeechToText
+def main():
+    amp_transcript =  sys.argv[1] 
+    words_to_flag_file =  sys.argv[2] 
+    output_csv =  sys.argv[3] 
+
+    # Get a list of words to flag
+    words_to_flag = get_words(words_to_flag_file) 
+    print("Words to Flag:")
+    for w in words_to_flag:
+        print(w)
+    print("")
+
+    # Search for matching words/phrases
+    matching_words = list()
+    with open(amp_transcript, 'r') as f:
+        transcript_dict = json.load(f)
+        # Check to see if we have a valid transcript.  If so, find the matches. 
+        if "results" in transcript_dict.keys() and "words" in transcript_dict["results"].keys():
+            transcript_words = transcript_dict["results"]["words"]
+            matching_words = match_words(words_to_flag, transcript_words)
+
+    # Print the output
+    print(matching_words)
```

Comment:
print "Matching words:" or such before the matching_words

Reply:
Added!